Solutions/Recorded Future/Playbooks/readme.md

```diff
@@ -0,0 +1,71 @@
+# RecordedFuture - Import Indicators Logic Apps and IOC Enrichment Playbook templates
+
+## Table of Contents
+
+1. [Overview](#overview)
+1. [Import To Sentinel - Deployment](#deployimporttosentinel)
+1. [IP - Actively Communicating C&C Server - Indicator Processor - Deployment](#deployIPindicatorprocessor)
+1. [Domain - C&C DNS Name - Indicator Processor - Deployment](#deployDOMAINindicatorprocessor)
+1. [URL - Recently Reported by Insikt Group - Indicator Processor - Deployment](#deployURLindicatorprocessor)
+1. [HASH - Observed in Underground Virus Test Sites - Indicator Processor - Deployment](#deployHASHindicatorprocessor)
+1. [IOC (IP, Domain, URL, Hash) Enrichment Playbook - Deployment](#deployIOCEnrichment)
+
+<a name="overview">
+
+# Overview
+
+Recorded Future is the world’s largest provider of intelligence for enterprise security. By combining persistent and pervasive automated data collection and analytics with human analysis, Recorded Future delivers intelligence that is timely, accurate, and actionable.
+
+<a name="deployimporttosentinel">
+
+# Deployment
+
+Due to internal Microsoft Logic Apps dependencies, please deploy first the ImportToSentinel playbook before any of the IndicatorProcessor one. This playbooks will serve via the Microsoft Batching mechanism all of the IndicatorProcessor playbooks, for optimizition of the indicator deployment process.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FRecorded%20Future%2FPlaybooks%2FRecordedFuture-ImportToSentinel.json)
```

Comment:
Deploy to azure link is not working. Image for reference.
![image](https://user-images.githubusercontent.com/87809732/134477908-879981c6-7daf-4229-aab2-57fd262185ca.png)